tests/test_squeues.py

```diff
@@ -31,6 +32,8 @@ def test_serialize(self):
         self.assertEqual(q.pop(), {'a': 'dict'})
 
     def test_nonserializable_object(self):
+        # Trigger Twisted bug #7989
+        import twisted.persisted.styles  # NOQA
```

Comment:
considering that the module is already imported at the top, won't this line be a no-op?